command/genesis/genesis.go

```diff
@@ -179,6 +179,15 @@ func setFlags(cmd *cobra.Command) {
 			[]string{},
 			"validators defined by user throughout a parameter (format: <address>:<blskey>)",
 		)
+		cmd.Flags().StringVar(
+			&params.premineValidators,
+			premineValidatorsFlag,
+			command.DefaultPremineBalance,
+			"the amount which will be premined to all the validators",
+		)
```

Comment:
Why do you need a special flag for this?

There is already a `premine` flag that you can utilize

Reply:
I wonder the same question, but the premine flag should be in the `format: <address>:<balance>` it's for specific addresses but not the validators addresses.

Do you have a suggestion on how to reuse the existing premine flag?

Reply:
Lets agree to remove this as soon as we finish all the child tasks if not needed. It makes our life easier while transferring code from v3 to the edge because we do not need to write any additional scripts